src/classes/CON_ContactMerge_TEST2.cls

```diff
@@ -880,4 +817,103 @@ private class CON_ContactMerge_TEST2 {
         }
         
     }
+
+        /*******************************************************************************************************
+        * @description Merge two Contacts where the master has no affiliations and the merged contact has a
+        ** primary affiliation.
+        **/
+        @isTest
+        static void mergeContactsWithAffiliation() {
+        
+            Account company = new Account(name = 'Foo Company');
+            insert company;
+
+            List<Contact> objectsToInsert = new List<Contact>();
+            Contact con = UTIL_UnitTestData_TEST.getContact();
+            con.LastName = '1Contact_forTests_Merge';
+            con.Description = 'First Contact';
+
+            Contact con2 = UTIL_UnitTestData_TEST.getContact();
+            con2.LastName = '2Contact_forTests_Merge';
+            con2.Description = 'Second Contact';
+            con2.Primary_Affiliation__c = company.Id;
+
+            objectsToInsert.add(con);
+            objectsToInsert.add(con2);
+
+            insert objectsToInsert;
+            
+            List<Contact> contactsToMerge = [
+                SELECT Id, AccountId, FirstName, LastName, MailingStreet, Description , Primary_Affiliation__c
+                FROM Contact 
+                ORDER BY Id
+            ];
+            con = contactsToMerge[0];
+            con2 = contactsToMerge[1];
+
+            List<npe5__Affiliation__c> affiliations = [
+                SELECT Id, npe5__Contact__c, npe5__EndDate__c, npe5__Organization__c, 
+                    npe5__Primary__c, npe5__Role__c, npe5__StartDate__c, npe5__Status__c, npe5__Description__c
+                FROM npe5__Affiliation__c 
+                WHERE npe5__Contact__c IN :contactsToMerge
+            ];
+
+            System.assertEquals(1, affiliations.size(), 'Affiliations are created for the contact');
+            
+            // Test.setCurrentPageReference(new PageReference('Page.CON_ContactMerge'));
```

Comment:
Do we need this comment?

Reply:
No we don't.  I forgot to remove it after I refactored the test function to include it.  I've removed it now. Thanks